.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -5053,6 +5053,10 @@ stages:
         env:
           DD_API_KEY: $(SYSTEM_TESTS_DD_API_KEY)
           TEST_LIBRARY: dotnet
+          AWS_ACCESS_KEY_ID: $(SYSTEM_TESTS_IDM_AWS_ACCESS_KEY_ID)
+          AWS_SECRET_ACCESS_KEY: $(SYSTEM_TESTS_IDM_AWS_SECRET_ACCESS_KEY)
+          AWS_REGION: us-east-1
```

Comment:
I think this one is actually overridden by a current key that was already set, but not set to be overridable - I've updated it to be overridable so it should work 👍

Reply:
Actually disregard that, I think that is only for being able to override it when running manually